### PR TITLE
chore: localeLoader: allow manually select which locale to load

### PR DESCRIPTION
**Pull request by undefined-moe**

### CODE REVIEW

File: packages/wrapper/src/vscode/localeLoader.ts [hunk 1]
```diff
@@ -59,3 +60,5 @@ export const initLocalLoader = async () => {
         }
     }
 };
+/** @deprecated use `initLocaleLoader` instead */
+export const initLocalLoader = initLocaleLoader;
```
Comment (kaisalmen) [inline packages/wrapper/src/vscode/localeLoader.ts:64]: Thank you for spotting the typo in the function name. There is no need to deprecate the old name, just remove it.

**Review by CGNonofr — COMMENTED** [review]
Looks like a good idea indeed!

**Review by kaisalmen — CHANGES_REQUESTED** [review]
Thank you very much. No need to deprecate the wrong function name. Just drop that and then I will approve this. 🎉

File: packages/wrapper/src/vscode/localeLoader.ts (hunk 1 above)
Comment (undefined-moe) [reply packages/wrapper/src/vscode/localeLoader.ts]: I've got that updated 👍

**Review by kaisalmen — APPROVED** [review]
LGTM 🎉

**Pull request merged**